performance/src/main/java/org/hibernate/validator/performance/cascaded/CascadedValidation.java

```diff
@@ -65,20 +66,24 @@ public void testCascadedValidation(CascadedValidationState state) {
 
 		Set<ConstraintViolation<Person>> violations = state.validator.validate( kermit );
 		assertThat( violations ).hasSize( 0 );
+
+		bh.consume( violations );
 	}
 
+/*
```

Comment:
Sorry, my mistake. I intended to remove this test, the for loop is completely optimised away by the JVM, and the test JMH test does not perform any actual work. If a large number of iterations is required as a test, then new Benchmark method with 50 * NUMBER_OF_VALIDATION_ITERATIONS would be appropriate and remove the for loop.  This would allow the JMH framework to manage the number of iterations.